### ISSUE TITLE
Object.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by andya9** [nesQuick/atom-wordcount#31]
Requiring `View` from `atom` is no longer supported.
Please require `atom-space-pen-views` instead:
  `{View} = require 'atom-space-pen-views'`
Add `"atom-space-pen-views": "^2.0.3"` to your package dependencies.
```
Object.Object.defineProperty.get (/usr/share/atom/resources/app/exports/atom.js:61:9)
Object.<anonymous> (/data/linux/configs/atom/packages/wordcount/lib/wordcount-view.coffee:1:1)
```

**Comment by nesQuick** [comment]
I'm happy to take your pull requests to get this fixed :)

**Comment by nesQuick** [comment]
fixed with 2.2.2
